examples/gocean/eg5/Makefile

```diff
@@ -31,127 +31,8 @@
 # ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # ------------------------------------------------------------------------------
-# Author: J. Henrichs, Bureau of Meteorology
-# Modified: A. R. Porter, STFC Daresbury Laboratory
+# Author: . Henrichs, Bureau of Meteorology
```

Comment:
Extraneous "."

Reply:
No, missing 'J' :grin:

Reply:
Oh yes, don't know how I didn't see that!